virtual_rainforest/core/config.py

```diff
@@ -379,5 +475,5 @@ def validate_config(
     with open(f"{output_folder}/{out_file_name}.toml", "wb") as toml_file:
         tomli_w.dump(config_dict, toml_file)
 
-    # Populate the global config dictionary with the complete validated config
-    COMPLETE_CONFIG["config"] = config_dict
+    # Return the complete validated config
+    return config_dict
```

Comment:
As above, don't bother returning `config_dict` again if it's actually modified in place.

If you *don't* want it to be modified in place, it might be worth having a test that checks whether this holds true.

Reply:
Agree with the above comment, but in this case `config_dict` is created within the function, so I think it has to be returned to be accessible to downstream functions?